content/docs/howto/negate-a-disjunction/en.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Negate a disjunction
+tags:
+- validation
+- language
+- commented cue
+authors:
+- hansbogert
+relatedIssues:
+- https://github.com/cue-lang/cue/discussions/3229
+- https://github.com/cue-lang/docs-and-content/issues/160
+toc_hide: true
+---
+
+This [Commented CUE]({{< relref "docs/howto#commented-cue-guides" >}})
+demonstrates how to negate a disjunction.
+
+{{{with code "en" "example"}}}
+! exec cue vet example.cue
+cmp stderr out
+-- example.cue --
+package example
+
+// Define a disjunction
+#oneTwoThree: "one" | "two" | "three"
+
+// Define a concrete value
+four: "four"
```

Comment:
I'd prefer to use data field names `A` and `B`. This will distinguish their names from their values, and prevent a user making the inference that names and values have to be aligned.